src/custom_fourier_types.jl

```diff
@@ -1,20 +1,22 @@
 ##  This View checks for the index to be L1 or the mirrored version (L2)
 # and then replaces the value by half of the parent at L1
+# do_split is a Bool that indicates whether this mechanism is active. It is needed for type stability reasons of functions returnting this type
```

Comment:
`do_split`.
Always wrap code in single ` `

Reply:
done